pages/star-search/waitlist.tsx

```diff
@@ -155,21 +191,36 @@ export default function StarSearchWaitListPage({
               StarSearch
             </h1>
           </div>
-          <p className="font-semibold text-5xl text-center tracking-tight text-balance">Copilot, but for git history</p>
-          <p className="text-center">Ask anything, get AI powered insights based on contributor data</p>
+          <p className="font-semibold text-5xl text-center tracking-tight text-balance">
+            {getTitle({ sessionToken, isWaitlisted, waitlistCount, alreadyOnWaitlist })}
+          </p>
+          <p className="text-center">
+            {isWaitlisted
+              ? "Share with others to get early access sooner"
+              : "Ask anything, get AI powered insights based on contributor data"}
+          </p>
           <div className="grid place-content-center h-16">
             {sessionToken && isWaitlisted && waitlistCount ? (
-              <>
-                {alreadyOnWaitlist ? (
-                  <p className="grid place-content-center">
-                    You&apos;re already on the waitlist along with {humanizeNumber(waitlistCount)} other people.
-                  </p>
-                ) : (
-                  <p className="grid place-content-center">
-                    You&apos;re in along with {humanizeNumber(waitlistCount)} other people on the StarSearch waitlist!
-                  </p>
-                )}
-              </>
+              <div className="flex gap-4">
+                <Button
```

Comment:
If we plan to do other social share buttons for these platform in the future, we can create reusuable social share buttons.